app/background.h

```diff
@@ -0,0 +1,21 @@
+/* This is licensed software, @see LICENSE file.
+ * Authors - @see AUTHORS file. */
+
+#ifndef _BACKGROUND_H_
+#define _BACKGROUND_H_
+
+#include <opencv2/core/mat.hpp>
+
```

Comment:
Forward-declare `struct background_t;` to pass typed pointers to all those functions. Doesn't affect actual API/structure for this structure on actual definition.

Reply:
Ah yep, will fix.